presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -1128,6 +1128,33 @@ public void testCreateTable()
                         true,
                         ImmutableList.of(),
                         Optional.of("test")));
+
+        assertStatement("CREATE TABLE foo (a VARCHAR, b BIGINT COMMENT 'hello world', c IPADDRESS)",
+                new CreateTable(QualifiedName.of("foo"),
+                        ImmutableList.of(
+                                new ColumnDefinition(identifier("a"), "VARCHAR", emptyList(), Optional.empty()),
+                                new ColumnDefinition(identifier("b"), "BIGINT", emptyList(), Optional.of("hello world")),
+                                new ColumnDefinition(identifier("c"), "IPADDRESS", emptyList(), Optional.empty())),
+                        false,
+                        ImmutableList.of(),
+                        Optional.empty()));
```

Comment:
I don't understand why this test case, as written, is related to this change. I presume you intended to use reserved words as column names.

Reply:
Removed